platform_versioned_docs/version-24.1.0/credentials/managed_identities.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Managed identities"
+description: "Configure managed identities in Seqera Platform."
+date: "12 Jun 2024"
+tags: [organizations, administration, managed identities, clusters, ssh, credentials]
+---
+
+With traditional SSH credentials in Seqera Platform, all workspace users access HPC clusters with the same set of credentials configured in the workspace. While this simplifies access control to computing resources, all user activity on your HPC cluster is logged under the same user credentials. This also means that all Seqera workspace users have the same access permissions on your HPC cluster. When multiple workspace users run pipelines and consume compute resources, all activity is reflected under the same user on your HPC cluster. Seqera managed identities resolve this limitation by enabling granular access control to HPC clusters for individual workspace users. 
+
+Configure a managed identity at the organization level to be used to access the supported HPC compute environments in your organization workspaces. A managed identity is selected for authentication to your compute environment in the same way that a traditional credential would be selected, but it contains multiple user credentials each tied to a unique Seqera user. This preserves the identity of the user launching workflows on your HPC cluster or compute environment, thereby improving traceability and adherence to data access policies. 
+
+By grouping individual user SSH credentials into a single Seqera element, managed identities allow administrators to streamline user login and compute environment access while maintaining visibility into data access and compute resource usage for each user. 
+
+## Create a managed identity 
+
+Organization owners can create managed identities at the organization level. A managed identity with user credentials can be used as a credential in HPC clusters for the same provider. 
+
+1. From your organization page, select the **Managed identities** tab, then **Add managed identity**.
+1. Enter the details of your cluster:
+    - A unique **Cluster name** of your choice using alphanumeric, dash, and underscore characters. 
+    - Select a cluster **Provider** from the dropdown.
+    - The fully qualified cluster **Hostname** to be used to connect to the cluster via SSH. This is usually the cluster login node. 
+    - The SSH **Port** number for the login connection. The default is port 22. 
+1. Select **Add cluster**. The new cluster is now listed under your organization's managed identities. 
+
+Select **Edit** next to a managed identity in the list to edit its details and add user credentials. 
+
+:::note
+If the managed identity is already in use on a compute environment, editing its details may lead to errors when using the compute environment. 
```

Comment:
Are there remediation steps for this or an alternative solution?

Reply:
No, just a warning not to edit clusters already in use, unless strictly necessary and the options being edited are correct on the HPC side as well.